blak05-cv08

krystofblak05 opened the pull request:
Snad vše, co jsme stihli na cvičení. Takže ještě nefunguje košík a přihlašování je jen přes cookie.
https://eso.vse.cz/~blak05/cv08/

krystofblak05 committed:
blak05-cv08

Snad vše, co jsme stihli na cvičení. Takže ještě nefunguje košík a přihlašování je jen přes cookie.
https://eso.vse.cz/~blak05/cv08/

nvbach91 reviewed (changes requested):
chybi edit, add a delete, delete je easy, mozna to muzete zkusit

krystofblak05 committed:
blak05-cv08

Přidal jsem delete a edit, a funguje i košík z dnešního cvika. Ostatní věci přidám do nového requestu cv09.
